docs/openapi/api.yaml

```diff
@@ -77,6 +79,10 @@ paths:
     $ref: './audits-api.yaml#/audits-for-site'
   /sites/{siteId}/latest-audit/{auditType}:
     $ref: './audit-api.yaml#/latest-audit-for-site'
+  /slack/create:
+    $ref: './slack-api.yaml#/create'
+  /slack/invite:
```

Comment:
also seems too generic, as the invitation is bound to an IMS Org... `inviteToChannelByIMSOrg` or similar?

Reply:
I agree `invite` is too generic, especially since you can invite to a channel, to a workspace etc. The detailed naming should align with the existing dash-case used in paths.
What about `POST /slack/channels/invite` to invite to a channel?